yarn.lock

```diff
@@ -1217,6 +1217,24 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@fluentui-react-native/framework@0.7.0":
+  version "0.7.0"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/framework/-/framework-0.7.0.tgz#328aa95e191054421a89469f0f03eb6839e4603c"
+  integrity sha512-IAdqltmjTq740RUDWutZ/JkAuFc7cDOtsuG/4zK1zAyxnVaf824OtWMSKceEI8oKKClUocZ7WemgZs/RKBMcnw==
+  dependencies:
+    "@fluentui-react-native/composition" ">=0.7.0 <1.0.0"
+    "@fluentui-react-native/default-theme" ">=0.9.0 <1.0.0"
+    "@fluentui-react-native/immutable-merge" "^1.1.4"
+    "@fluentui-react-native/memo-cache" "^1.1.5"
+    "@fluentui-react-native/merge-props" ">=0.4.0 <1.0.0"
+    "@fluentui-react-native/theme-types" ">=0.13.0 <1.0.0"
+    "@fluentui-react-native/tokens" ">=0.11.0 <1.0.0"
+    "@fluentui-react-native/use-slot" ">=0.2.0 <1.0.0"
+    "@fluentui-react-native/use-slots" ">=0.6.0 <1.0.0"
+    "@fluentui-react-native/use-styling" ">=0.8.0 <1.0.0"
+    "@fluentui-react-native/use-tokens" ">=0.2.0 <1.0.0"
+    tslib "^1.13.0"
+
```

Comment:
I think something is referencing an older package, hence this got added

Reply:
I'll check again after updating uifabricshared to @fluentui-react-native/scripts